e2e/__tests__/detectOpenHandles.ts

```diff
@@ -72,9 +72,13 @@ it('does not report promises', () => {
 });
 
 describe('notify', () => {
-  skipSuiteOnWindows();
-
   it('does not report --notify flag', () => {
+    if (process.platform === 'win32') {
```

Comment:
Isn't this the same as the skip?

Reply:
Because the skip causes *all* tests in the file to be skipped, but this is the only one that actually fails.